b3d/pose.py

```diff
@@ -239,6 +265,12 @@ def concatenate_poses(pose_list):
             jnp.concatenate([pose.quat for pose in pose_list], axis=0),
         )
 
+    def concat(self, poses, axis=0):
```

Comment:
I don't see this used anywhere -- does this work?  Or does this code need to expand the dimension of `self.pos` and `self.quat` before we concatenate, here?

(Happy to merge and fix this if we use it and it errors later.)